apps/greencheck/management/commands/dump_green_domains.py

```diff
@@ -23,62 +36,141 @@ def get_conn_string(self):
         # change this if we ever leave mysql, obvs
         return f"mysql://{db['USER']}:{db['PASSWORD']}@{db['HOST']}/{db['NAME']}"
 
-    def export_to_sqlite(self, database_url: str, db_name: str):
-        return subprocess.run(
-            ["db-to-sqlite", database_url, db_name, "--table=greendomain"]
+    @classmethod
+    def export_to_sqlite(cls, database_url: str, db_path: str) -> None:
+        """
+        Export the `cls.TABLE` to `db_path` from given `database_url`.
+
+        :param database_url: The database URL to export the `cls.TABLE` from.
+        :param db_path: The path to file to export SQLite database to.
+        """
+        cls._subprocess(
+            ["db-to-sqlite", database_url, db_path, f"--table={cls.TABLE}"],
+            f'Failed to export the "{cls.TABLE}" table to "{db_path}" SQLite database.'
         )
 
-    def prepare_for_datasette(self, db_name: str):
+    @classmethod
+    def prepare_for_datasette(cls, db_path: str) -> None:
         """
         Make sure we have the indexes we need for use in datasette
         """
-        db = Database(db_name)
-        green_domains_table = db["greendomain"]
+        db = Database(db_path)
+        green_domains_table = db[cls.TABLE]
         # green_domains_table.create_index(["hosted_by"])
 
-    def compress_file(self, file_path: str):
-        """
-        Compress the file at file_path with gzip.
-        Should compress a sqlite file down to ~25% of original size.
+    @classmethod
+    def compress_file(cls, file_path: str, compression_type: str = "gzip") -> str:
         """
-        subprocess.run(["gzip", "--force", file_path])
+        Compress the file at `file_path` with `gzip` or `bzip2`.
 
-    def delete_file(self, file_path: str):
-        subprocess.run(["rm", "-rf", file_path])
-
-    def upload_file(self, file_path: str, file_destination: str):
-        subprocess.run(["aws", "s3", "cp", "--acl", "public-read", file_path, file_destination])
+        :param file_path: The path to file to compress.
+        :param compression_type: The one of compression types.
+        :returns: The path to created archive.
+        :raises Exception: When `compression_type` is invalid.
+        :raises RuntimeError: When the compression process fails.
+        """
+        if compression_type not in _COMPRESSION_TYPES:
+            raise Exception((
+                f'The "{compression_type}" compression is not supported. Use '
+                'one of "%s".' % ('", "'.join(_COMPRESSION_TYPES.keys()))
```

Comment:
Would you assign this with a descriptive variable name, and then include it using f-strings like you have elsewhere please? Everything else in this PR is really readable, but I found this harder to scan when reading the code.